Cargo.toml

```
@@ -1,20 +1,20 @@
 [package]
 edition = "2021"
 name = "bzip2"
-version = "0.4.4"
-authors = ["Alex Crichton <alex@alexcrichton.com>"]
+version = "0.5.0"
 license = "MIT/Apache-2.0"
```

```suggestion
license = "MIT OR Apache-2.0"
```
This is the correct syntax:
> If 2 or more licenses apply to a file, use an SPDX license expression. It is a composite expression constructed using parentheses, AND / OR operators, and the WITH operator for license exceptions (see below).